solutions/devicesimulation-nohub/single-vm/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '2'
+
+services:
+  reverseproxy:
+    image: azureiotpcs/simulation-api-gateway:millennium
+    ports:
+      - "80:80"
+      - "443:443"
+    depends_on:
+      - webui
+      - devicesimulation
+    volumes:
+      - /app/certs:/app/certs:ro
+
+  webui:
+    image: azureiotpcs/device-simulation-webui:millennium
+    depends_on:
+      - devicesimulation
+    volumes:
+      - /app/webui-config.js:/app/build/webui-config.js:ro
+
+  devicesimulation:
+    image: azureiotpcs/device-simulation-dotnet:millennium
+    depends_on:
+      - storageadapter
+    environment:
+      - PCS_IOTHUB_CONNSTRING
```

Comment:
This environment variable was not initialized in script setup.sh. What value will the simulation service will get?

Reply:
will set to "none"